index.js

```diff
@@ -8,10 +9,24 @@ const parser = require("./src/parser");
 const printDocToDebug = require("./src/doc-debug").printDocToDebug;
 
 function parse(text, opts) {
-  if (opts.parser === "flow") {
-    return parser.parseWithFlow(text, opts.filename);
+  const parseFunction = opts.parser === "flow"
+    ? parser.parseWithFlow
+    : parser.parseWithBabylon;
+
+  try {
+    return parseFunction(text);
+  } catch (error) {
+    const loc = error.loc;
+
+    if (loc) {
+      error.codeFrame = codeFrame(text, loc.line, loc.column + 1, {
+        highlightCode: true
```

Comment:
@existentialism What did you do to take that screen shot? When running `npm run build:docs` and then opening docs/index.html I get `ReferenceError: process is not defined` pointing to a line saying `var argv = process.argv;`.